src/public/js/github/github.js

```diff
@@ -0,0 +1,13 @@
+import orgs from './orgs.js';
+import repos from './repos.js';
+import issues from './issues.js';
+
+export default (view, payload) => {
+  if (view === 'orgs') {
+    orgs(payload);
```

Comment:
i feel that we should have the organisations, repos, and timer in the same 'view'. When we drew out the basic wireframe that first day at FAC they were all on the same page so that you could edit one without having to 'go back'. I'll try and find the wireframe to show you, but how do you feel about that in general?